lib/SqlFormatter.php

```diff
@@ -122,6 +122,13 @@ class SqlFormatter
     // If not defined, it will be determined automatically
     public static $cli;
 
+    // Comments
+    public static $comment_tokens = [
```

Comment:
Vyrazy pre komentare je mozne uz definovat, ... ak je iba jedna polozka, tak sa berie, ze je to riadkovy komentar, ... inak cely blok.